conviva/src/main/java/com/bitmovin/analytics/conviva/ConvivaAnalyticsIntegration.java

```diff
@@ -230,95 +176,102 @@ public void reportPlaybackDeficiency(String message, Client.ErrorSeverity severi
-        if (!isSessionActive()) {
-            return;
-        }
-
         String eventName = event.getClass().getSimpleName();
         sendCustomPlaybackEvent("on" + eventName, attributes);
     }
 
     // region Session handling
     private void setupPlayerStateManager() {
-        try {
-            playerStateManager = client.getPlayerStateManager();
-            playerStateManager.setPlayerState(PlayerStateManager.PlayerState.STOPPED);
-            playerStateManager.setPlayerType("Bitmovin Player Android");
-            playerStateManager.setPlayerVersion(playerHelper.getSdkVersionString());
-            clientMeasureInterface = new ClientMeasureInterface();
-            playerStateManager.setClientMeasureInterface(clientMeasureInterface);
-        } catch (ConvivaException e) {
-            Log.e(TAG, e.getLocalizedMessage());
-        }
+        convivaVideoAnalytics.reportPlaybackMetric(ConvivaSdkConstants.PLAYBACK.PLAYER_STATE, ConvivaSdkConstants.PlayerState.STOPPED);
+        Map<String, Object> playerInfo = new HashMap<String, Object>();
+        playerInfo.put(ConvivaSdkConstants.FRAMEWORK_NAME,"Bitmovin Player Android");
+        playerInfo.put(ConvivaSdkConstants.FRAMEWORK_VERSION, playerHelper.getSdkVersionString());
+        convivaVideoAnalytics.setPlayerInfo(playerInfo);
     }
 
     private void internalInitializeSession() {
-        try {
-            createContentMetadata();
-            sessionId = client.createSession(contentMetadataBuilder.build());
-            setupPlayerStateManager();
-            if (metadataOverrides != null) {
-                updateContentMetadata(metadataOverrides);
-            }
-            Log.d(TAG, "[Player Event] Created SessionID - " + sessionId);
-            client.attachPlayer(sessionId, playerStateManager);
-        } catch (ConvivaException e) {
-            Log.e(TAG, e.getLocalizedMessage());
+        if(isSessionActive) {
+            return;
+        }
+        Log.d(TAG, "internalInitializeSession");
+        createContentMetadata();
+        convivaVideoAnalytics.reportPlaybackRequested(contentMetadataBuilder.build());
+        setupPlayerStateManager();
+        if (metadataOverrides != null) {
+            updateContentMetadata(metadataOverrides);
         }
+        isSessionActive = true;
     }
 
     private void updateSession() {
-        if (!isSessionActive()) {
-            return;
-        }
         this.buildDynamicContentMetadata();
 
         VideoQuality videoQuality = bitmovinPlayer.getPlaybackVideoData();
         if (videoQuality != null) {
             int bitrate = videoQuality.getBitrate() / 1000; // in kbps
-            try {
-                playerStateManager.setBitrateKbps(bitrate);
-                playerStateManager.setVideoHeight(videoQuality.getHeight());
-                playerStateManager.setVideoWidth(videoQuality.getWidth());
-                clientMeasureInterface.setFrameRate(videoQuality.getFrameRate());
```

Comment:
Since now ClientMeasureInterface is removed, how is frame rate information updated for the playback session?

Reply:
This info is actually still sent in `updateSession()`: 
```
        VideoQuality videoQuality = bitmovinPlayer.getPlaybackVideoData();
        if (videoQuality != null) {
            int bitrate = videoQuality.getBitrate() / 1000; // in kbps
                convivaVideoAnalytics.reportPlaybackMetric(ConvivaSdkConstants.PLAYBACK.RESOLUTION, videoQuality.getHeight(), videoQuality.getWidth());
                convivaVideoAnalytics.reportPlaybackMetric(ConvivaSdkConstants.PLAYBACK.BITRATE, bitrate);
                convivaVideoAnalytics.reportPlaybackMetric(ConvivaSdkConstants.PLAYBACK.RENDERED_FRAMERATE, Math.round(videoQuality.getFrameRate()));
        }
```

Reply:
Ah..thanks. Sorry missed that earlier.